backend/services/auth/app/api/v1/financial_reports.py

```
@@ -0,0 +1,315 @@
+
+        result = await financial_service.generate_financial_summary(
+            period=period,
+            start_date=start_date,
+            end_date=end_date,
+            fleet_id=fleet_id,
+            db=db,
+        )
+
+        return FinancialSummaryResponse(**result)
+
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)